### DISCUSSION TITLE
Built in support for Starter Assets

### DISCUSSION THREAD

**Discussion by vinay-wowlabz** [readyplayerme/rpm-unity-sdk-core#60] [Ideas]
Hi,
Right now, we're able to integrate RPM with Unity's Starter Assets using some custom code. Is there a plan to release a version that automatically has the Starter Assets system integrated with the Avatar? This would be really useful

**Comment by srcnalt** [comment]
Hi @vinay-wowlabz, currently we have a Quick Start sample that comes with the Avatar Loader package of SDK, which provides a basic use of Third Person camera. There is no plan to make a direct Starter Assets integration at the moment but probably we can help out if you are having any integration issues.
